src/assets.zig

```diff
@@ -271,21 +273,35 @@ fn registerItem(assetFolder: []const u8, id: []const u8, zon: ZonElement) !*item
 		texturePath = try std.fmt.bufPrint(&buf1, "{s}/{s}/items/textures/{s}", .{assetFolder, mod, texture});
 		replacementTexturePath = try std.fmt.bufPrint(&buf2, "assets/{s}/items/textures/{s}", .{mod, texture});
 	}
-	return items_zig.register(assetFolder, texturePath, replacementTexturePath, id, zon);
+	_ = items_zig.register(assetFolder, texturePath, replacementTexturePath, id, zon);
```

Comment:
We should probably remove all the unnecessary return values from these, but that should be done in another PR.